package.json

```diff
@@ -25,6 +25,10 @@
     "oracle-e2e": "./oracle-e2e/run-tests.sh",
     "ui-e2e": "./ui-e2e/run-tests.sh",
     "clean": "rm -rf ./node_modules ./**/node_modules ./**/**/node_modules ./**/build",
+    "compile:contracts": "yarn workspace poa-parity-bridge-contracts run compile",
```

Comment:
Just for you information: I have created https://github.com/poanetwork/poa-bridge-contracts/issues/189 to change the legacy package name.